infrastructure/haproxy_reload.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
-#:80
-real=0
+#:80/443
+http=0
 #:81
 stats=1
 
+DPORT=80
+[ ${HAPROXY_SSL} == "true" ] && DPORT=443 && export CERT_OPTS="${HAPROXY_CERT_OPTS}"
```

Comment:
actually, `HAPROXY_CERT_OPTS` will be set to `ssl crt /etc/haproxy/haproxy.pem` by default in `generate_yml.sh`.  we wouldn't want this to go into the template unless `HAPROXY_SSL=true`.